hatch_build.py

```diff
@@ -225,6 +225,8 @@
     "itsdangerous>=2.0",
     "jinja2>=3.0.0",
     "jsonschema>=4.18.0",
+    # Added for CLI
```

Comment:
```suggestion
```

Reply:
I added not to forget while moving out CLI but we can identify at that time. Removed.